examples/Targeting/SearchForTargetableLanguagesAndCarriers.cs

```diff
@@ -0,0 +1,167 @@
+using Google.Ads.GoogleAds.V3.Errors;
+using Google.Ads.GoogleAds.V3.Common;
+using Google.Ads.GoogleAds.V3.Resources;
+using Google.Ads.GoogleAds.V3.Services;
+
+using System;
+
+using static Google.Ads.GoogleAds.V3.Enums.KeywordMatchTypeEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.ProximityRadiusUnitsEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    /// <summary>
+    /// This code example illustrates how to search for language constants by name that
+    /// includes the specified keyword. Then it searches for all the available mobile carrier
+    /// constants with a given country code.
+    /// </summary>
+    public class SearchForTargetableLanguagesAndCarriers : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            SearchForTargetableLanguagesAndCarriers codeExample =
+                new SearchForTargetableLanguagesAndCarriers();
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            // The keyword to search for.
+            string searchKeyword = "INSERT_SEARCH_KEYWORD_HERE";
+
+            // The country code for which the search is performed.
+            // A list of country codes can be referenced here:
+            // https://developers.google.com/adwords/api/docs/appendix/geotargeting.
+            string carrierCountryCode = "US";
+
+            codeExample.Run(new GoogleAdsClient(), customerId, searchKeyword, carrierCountryCode);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example illustrates how to search for language constants by " +
+                    "name that includes the specified keyword. Then it searches for all the " +
+                    "available mobile carrier constants with a given country code.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
+        /// <param name="searchKeyword">The keyword to search for language code.</param>
+        /// <param name="carrierCountryCode">The country code to search for carrier code.</param>
+        public void Run(GoogleAdsClient client, long customerId, string searchKeyword,
+            string carrierCountryCode)
+        {
+            try
+            {
+                SearchTargetableLanguages(client, customerId, searchKeyword);
```

Comment:
The golden does everything in the main, better to keep it consistent across the client libs.

Reply:
I think these are two different unrelated API calls, and it is better to keep them as separate methods.

Reply:
SG